2021/utils.py

```diff
@@ -0,0 +1,5 @@
+def file_to_list(datapath):
+    with open(datapath, "r") as f:
+        data = [int(line.strip("\n")) for line in f.readlines()]
+
+    return data
```

Comment:
I generally advise to avoid "util" functions if you can. I understand why lots of people write them, but I usually view them as a bit of a cop out / lazy place to put code, instead of putting it somewhere that makes more sense.

Is this code truly a utility that anyone could use? What if we don't want to parse the data into a list of ints?

There's also the other side of this, where we aren't _really_ gaining anything from abstracting this into a separate file. We now have a magic method called `file_to_list` that takes in a class path, and does "something" to turn it into a list of who knows what.

Part of the problem is the ambiguous name, but the other side of it is that we're trying to make a generic method, that isn't truly generic, as it's specific to our day 1 problem for now.

---

I'd be more inclined to simply inline this logic in `day1.py` for now. Maybe we do refactor it out later, but as of this PR, I think we're attempting to organize our code too early, with the end result being logic that's hidden for no real gain.